### ISSUE TITLE
Provide onBeforeCreate callback

### ISSUE THREAD

**Issue by wintercounter** [JeremyRH/storybook-addon-code-editor#5]
I'm trying to make the editor work with https://github.com/remcohaszing/monaco-tailwindcss so we can provide auto-complete for our Tailwind classes. To make this work, I'd need to configure monaco before creating the editor instance, which is currently not possible as I see. I guess a callback before creating the instance would be sufficient enough to make this work, but not sure. Any ideas how can I make this work?

**Comment by JeremyRH** [comment]
In version `1.2.0` I added a `setupEditor` callback that has 2 parameters: `monaco` and `createEditor`. The return value can be an editor instance or `void`. `setupEditor` is called before the editor instance is created, right after monaco loads. You can create and return an editor instance in your `setupEditor` callback or it will be created immediately afterwards for you if you return `void`.

Check out the [README](https://github.com/JeremyRH/storybook-addon-code-editor#createliveeditstory) or [example story](https://github.com/JeremyRH/storybook-addon-code-editor/blob/f6e70e9ba651f81cc023fef9e6f38811bd787b95/example/src/Button/stories/Button.stories.tsx#L15) and let me know if you have questions.

**Comment by JeremyRH** [comment]
With the latest version (`2.0.1`), I removed `setupEditor` and added a more global way to configure things. [`setupMonaco`](https://github.com/JeremyRH/storybook-addon-code-editor/blob/687d50e31e92e7209d9dd3d2cd107613024b7f6b/README.md#setupmonaco).
I figured out a hacky way to get https://github.com/remcohaszing/monaco-tailwindcss 's worker file loaded. Not sure if it's working completely but at least there are no errors.
